src/background/actions/assessment-card-selection-action-creator.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { StoreNames } from 'common/stores/store-names';
+
+import * as TelemetryEvents from '../../common/extension-telemetry-events';
+import { getStoreStateMessage, Messages } from '../../common/messages';
+import { Interpreter } from '../interpreter';
+import { TelemetryEventHandler } from '../telemetry/telemetry-event-handler';
+import {
+    AssessmentCardSelectionPayload,
+    BaseActionPayload,
+    RuleExpandCollapsePayload,
+} from './action-payloads';
+import { CardSelectionActions as AssessmentCardSelectionActions } from './card-selection-actions';
```

Comment:
With this current import we're importing the existing card selection actions but just renaming them to match the new assessment card selection action names, instead of actually importing the new assessment card selection actions. This should be `
import { AssessmentCardSelectionActions } from 'background/actions/assessment-card-selection-actions';`. I think this is where a lot of the payload mixups are coming from- fixing this import should cause fastpass to start reporting the places where the payloads are wrong.

Reply:
Good catch!